test/functional/eval/input_spec.lua

```diff
@@ -196,6 +228,18 @@ describe('input()', function()
     eq('Vim(call):E118: Too many arguments for function: input',
        exc_exec('call input("prompt> ", "default", "file", "extra")'))
   end)
+  it('supports highlighting', function()
+    feed([[:call input({"highlight": "RainBowParens"})<CR>]])
+    wait()
```

Comment:
this wait should still be removed. In any realistic situation, `input()` will be invoked with one call, not with an unrelated event incidentally right after it.